Idno/Core/Config.php

```diff
@@ -202,14 +203,14 @@ function loadIniFiles()
                         $bucket = $host_parts[0];
                         $this->ini_config['aws_key'] = $_ENV['CLOUDCUBE_ACCESS_KEY_ID'];
                         $this->ini_config['aws_secret'] = $_ENV['CLOUDCUBE_SECRET_ACCESS_KEY'];
-                        $this->ini_config['bucket'] = $bucket;
-                        $this->ini_config['aws_region'] = $bucket == 'cloudcube-eu' ? 'eu-west-1' : 'us-east-1';
```

Comment:
there has either been a change to add a hyphen in cloudcube, or I missed one by only testing in the US